postprocessing/validation/compare-energies.py

```diff
@@ -38,8 +38,8 @@
     print("Relative difference")
     print(relative_difference)
 
-    relative_difference_smaller_eps = (
-        relative_difference.iloc[1:, :] < args.epsilon
+    relative_difference_larger_eps = (
+        relative_difference.iloc[1:, :] > args.epsilon
     ).values
-    if not np.all(relative_difference_smaller_eps):
+    if np.any(relative_difference_larger_eps):
```

Comment:
Would be good to see where the problems lie (and how big they are)

Reply:
If both energies are zero and we compute the relative difference it is (0 - 0) / 0 = NaN.
`NaN < eps` evaluates to false, while `! (NaN > eps)` evaluates to true ;-)